debian/rules

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 
 DISTRIBUTION := $(shell lsb_release -sc)
-VERSION := $(shell make --no-print-directory version)
+VERSION := $(shell cat VERSION)
 DEBVERSION = $(VERSION)-$(shell date +%Y%m%d-%H%M%S)-0~$(DISTRIBUTION)0
 
 %:
```

Comment:
This rules file should work but we could simplify things further by using the Debian golang tools (https://go-team.pages.debian.net/packaging.html).

This is not a blocker for this PR though, it is already making things a bit easier.

Reply:
We can use debian go tools *and* the version hack here (it works just fine to override  dh_gencontrol as we do as well).

I'm not sure how debian tools handle go modules however (and they seem to not immediately work with vendor/ - but I've not dug into it).

So still nothing to do about this on this PR :wink:

Reply:
I'd be in favor of copying things previously built to the debian package, otherwise builds end up producing the same artifact twice.

Reply:
As far as I know, Debian tools will refuse to fetch dependencies and will not use vendored dependencies. I don't see how we could use them except for very simple cases. Moreover, it would contribute to the sentiment that Debian packaging is complicated as it adds another layer people won't know how to customize.